client/src/components/ArticlePageMainContent/ArticlePageMainContent.js

```diff
@@ -14,6 +14,10 @@ const ContentContainer = styled.section`
     margin: 0.5rem 0;
   }
 
+  h2 {
+    margin-top: 1.5rem;
+  }
+
   h4 {
     margin: 0.5rem 0;
   }
```

Comment:
Think of a default page layout to shorten your files and to avoid unnecessary code.
Check this example:
https://github.com/manuelpapa/food-sharing-app/blob/c49e9898c9124b6b0ea65a36b2f220f41cc3c071/client/src/components/PageLayout.js

Reply:
Cool idea! Thanks!